db/rmWrapper.py

```diff
@@ -170,11 +170,12 @@ def submit_raid(self, gym, pkm, lvl, start, end, type, raid_no, capture_time, un
                 "UPDATE raid "
                 "SET level = %s, spawn = FROM_UNIXTIME(%s), start = %s, end = %s, "
                 "pokemon_id = %s, last_scanned = FROM_UNIXTIME(%s), cp = %s, "
-                "move_1 = %s, move_2 = %s "
```

Comment:
You removed this and a few more spaces which are actually required and things will break without them.